StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/AttributedLabel.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Label.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 5/8/23.
+//
+
+import Foundation
+import UIKit
+
+// Prefer `AttributedLabel` over `AttributedTextView` for single-line text.
+final class AttributedLabel: UILabel {
+
+    private let customFont: FinancialConnectionsFont
+    private let customTextColor: UIColor
+
+    init(font: FinancialConnectionsFont, textColor: UIColor) {
+        self.customFont = font
+        self.customTextColor = textColor
+        super.init(frame: .zero)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // UILabel with custom `lineHeight` via `NSParagraphStyle` was not properly
+    // centering the text, so here we adjust it to be centered.
```

Comment:
this explains why I introduced `AttributedLabel`...the test plan shows the difference too

Reply:
Well this comment + the fact that `ClickableLabelNew` (now `AttributedTextView`) does not support single line text (at least easily)